contracts/Gauge.sol

```diff
@@ -154,7 +245,16 @@ contract Gauge is IGauge {
                 1e18) / totalSupply());
     }
 
-    function earned(address account) public view returns (uint256) {
+    /** @notice earning for an account
+     *  @dev earning are based on lock duration and boost
+     *  @return amount of tokens earned
+     */
+    function earned(address account)
+        external
+        view
+        override(BaseGauge, IBaseGauge)
```

Comment:
never seen this syntax before, need to learn more about it.  What's the point of overriding the interface as well?